podcast-player/src/utils/api-utils.js

```diff
@@ -42,6 +45,7 @@ export async function getEpisodes(url) {
       title: parsedXml.rss.channel.title,
     },
   };
-  console.log(parsedXml);
+  console.log("Original:", parsedXml);
```

Comment:
noisy console logs, dead code. What's the state of this PR? Is it ready for review?